lib/lambda_ethereum_consensus/p2p/incoming_request_handler.ex

```diff
@@ -117,8 +119,46 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequestHandler do
     end
   end
 
-  def handle_req("beacon_blocks_by_range/2/ssz_snappy", _message_id, _message),
-    do: :not_implemented
+  def handle_req("beacon_blocks_by_range/2/ssz_snappy", message_id, message) do
+    with <<24, snappy_blocks_by_range_request::binary>> <- message,
```

Comment:
what does the 24 reference to?

Reply:
That's the length of the `BeaconBlocksByRangeRequest` SSZ Container.